src/atoms/anchors/anchor.tsx

```diff
@@ -13,6 +15,7 @@ export interface AnchorProps {
     children?: any;
     cssClassName?: string;
     external?: boolean;
+    icon?: keyof typeof Icons;
```

Comment:
How do we feel about that being a different issue? `Icons` was previously set up as the way it is now, I didn't change it for this PR. I have some other thoughts on the icon component set up as well and it would be easier to tackle all of that related stuff together.